src/components/PrintModal/PrintModal.js

```diff
@@ -18,6 +18,7 @@ import { Swipeable } from 'react-swipeable';
 import './PrintModal.scss';
 import Choice from '../Choice/Choice';
 import { FocusTrap } from '@pdftron/webviewer-react-toolkit';
+import setPrintQuality from 'src/apis/setPrintQuality';
```

Comment:
Instead of APIs, we can dispatch an action to set the print quality. 

`store.dispatch(actions.setPrintQuality(quality));`

Reply:
after discussed with Andrey, I replace this import function call to `onChange={e => store.dispatch(actions.setPrintQuality(e.target.value))}`